gentrap/src/main/scala/nl/lumc/sasc/biopet/pipelines/gentrap/Gentrap.scala

```diff
@@ -136,6 +136,20 @@ class Gentrap(val parent: Configurable)
     )
   )
 
+  override def biopetScript(): Unit = {
+    val validate = new ValidateAnnotation(this)
+    validate.refflatFile = annotationRefFlat.get
+    validate.gtfFile = fragmentsPerGene.map(_.annotationGtf)
```

Comment:
why? when I want only to use cufflinks, then I don't need the annotation to be in refflat format